Loading ambiente Python...
-------------------------------
Iniciando script Python...
Traceback (most recent call last):
  File "/mnt/nfs/home/vcunha/home/hcp_for_edp/assignment02/python/serial/main_sequential.py", line 3, in <module>
    import pandas as pd
  File "/home/vcunha/.local/lib/python3.10/site-packages/pandas/__init__.py", line 49, in <module>
    from pandas.core.api import (
  File "/home/vcunha/.local/lib/python3.10/site-packages/pandas/core/api.py", line 9, in <module>
    from pandas.core.dtypes.dtypes import (
  File "/home/vcunha/.local/lib/python3.10/site-packages/pandas/core/dtypes/dtypes.py", line 47, in <module>
    from pandas.errors import PerformanceWarning
  File "/home/vcunha/.local/lib/python3.10/site-packages/pandas/errors/__init__.py", line 6, in <module>
    import ctypes
  File "/modules/apps/python/3.10.1/lib/python3.10/ctypes/__init__.py", line 8, in <module>
    from _ctypes import Union, Structure, Array
ModuleNotFoundError: No module named '_ctypes'
